pkgs/development/python-modules/trackpy/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, numpy
+, scipy
+, six
+, pandas
+, pyyaml
+, matplotlib
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "trackpy";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "soft-matter";
+    repo = "trackpy";
+    rev = "v${version}";
+    sha256 = "01fdv93f6z16gypmvqnlbjmcih7dmr7a63n5w9swmp11x3if4iyq";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+    scipy
+    six
+    pandas
+    pyyaml
+    matplotlib
+  ];
+
+  checkInputs = [
+    pytest
+  ];
+
+  checkPhase = ''
+    # specifically needed for darwin
```

Comment:
How much of this is needed for darwin, and is it possible you can condition it on `stdenv.isDarwin`?

Reply:
fixed.